<?xml version="1.0"?>
<launch>
  <rosparam param="pegasus_mavros_namespaces">
    [
      "uav0",
      "uav1",
      "uav2"
    ]
  </rosparam>
  <rosparam param="pegasus_local_transforms">
    [
      [ "map_0", "base_link_0"],
      [ "map_1", "base_link_1"],
      [ "map_2", "base_link_2"]
    ]
  </rosparam>
  <param name="agents_hover_height" value="40"/>
  <param name="grid_size" value="5"/>
  <param name="param_depth_exit" value="12"/>
  <param name="param_c_power" value="0.7"/>
  <param name="param_retry_threshold" value="8"/>
  <param name="param_early_exit" value="4092"/>
  <param name="param_num_directions" value="8" />
  <node pkg="pegasus_ros" type="pegasus_planner.py" name="pegasus_planner" respawn="false" output="screen">
  </node>
</launch>
